src/simple-notifications.module.ts

```diff
@@ -1,4 +1,5 @@
 import {NgModule, ModuleWithProviders} from '@angular/core';
+import {BrowserAnimationsModule} from '@angular/platform-browser/animations';
```

Comment:
You should add the BrowserAnimationsModule to the @NgModule imports array, to have the desired effect.